diagnostic.md

```diff
@@ -27,7 +27,7 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+$base-font-size: (h1*1.30);
```

Comment:
No, this isn't correct. We're looking to define the font size for h1; in normal CSS, that would be
```
h1 {
  font-size: ____;
}
```
We've defined a SASS variable called `$base-font-size`, with some size, and we want to define the font size for h1 with respect to that variable. We write:
```
h1 {
  font-size: ($base-font-size * 1.3)
}
```